src/flb_plugin_proxy.c

```diff
@@ -66,6 +66,22 @@ static void flb_proxy_cb_flush(const void *data, size_t bytes,
 }
 
 
+static void flb_proxy_cb_exit(struct flb_output_plugin *instance,
+                              struct flb_plugin_proxy_context *context) 
+{
+    struct flb_plugin_proxy *proxy = (instance->proxy);
```

Comment:
Since you don't use the context, you should cast it to void:

```
(void) context;
```

(This is shown with the config in the proxy flush above)

Reply:
I was thinking to submit changes to implement a plugin exit handler which can accept context as well - (I suppose users might eventually end up needing this if they were to write a `stateful` multi instance plugin) - I know we discussed this earlier, but do you think this will be useful for others as well? If not, ill open a PR to just remove the unused parameter.

Reply:
> do you think this will be useful for others as well

I think it might be, yeah. I don't personally have a need for it in my go plugins, but others might. 

If you want to work on adding that, I say go ahead 👍